cholesky.jl

```diff
@@ -0,0 +1,101 @@
+using Enzyme
+using LinearAlgebra
+using Random
+using SparseArrays
+using Test
+using FiniteDifferences
+
+include("utils.jl")
+include("utils_check.jl")
```

Comment:
These files don't exist, moreover can you put this inside of test/internal_rules.jl (and use the testing framework).

Reply:
Yes. Will do.